### ISSUE TITLE
Drengars furious attacks reset

### ISSUE THREAD

**Issue by Oblivionn** [IroncladGames/SinsOfADarkAge#871]
While writing a drengar guide, i thought i found out an undocumented stat change to AP from drengars ult.
Reset hero doesn't reset the increase of stats from furious blades, it just disables the active part.
Normal drengar, level 15, no items, has the scaled stat points since i cant disable them, has level 4 furious attacks.
129ap
![2015-01-23_00005](https://cloud.githubusercontent.com/assets/7982804/5874757/68ca59cc-a359-11e4-9a05-23224966e858.jpg)
Reset, put back to max level, still 129ap
![2015-01-23_00001](https://cloud.githubusercontent.com/assets/7982804/5874732/205383d0-a359-11e4-9240-101f2dc12724.jpg)
Various levels of rage to show the increases
![2015-01-23_00002](https://cloud.githubusercontent.com/assets/7982804/5874733/27e4b20e-a359-11e4-95b2-85027e6ff9f4.jpg)
![2015-01-23_00003](https://cloud.githubusercontent.com/assets/7982804/5874734/2812917e-a359-11e4-92ff-297c131f9e48.jpg)
![2015-01-23_00004](https://cloud.githubusercontent.com/assets/7982804/5874735/2848efa8-a359-11e4-9b02-5589cf4abc43.jpg)

**Comment by xAzta** [comment]
Good catch, tested the same with Xeros E magic resist passive, and it does the same thing.
